packages/boxel-ui/addon/src/components/input-group/accessories/index.gts

```diff
@@ -211,6 +204,14 @@ export const Select: TemplateOnlyComponent<SelectAccessorySignature> =
         border-color: var(--boxel-error-100);
       }
     </style>
+    <style>
```

Comment:
Does this need to be in a separate `style` tag?

Reply:
No it doesn